samples/xti/xti.cpp

```diff
@@ -277,13 +277,13 @@ class MyDesignerPersister : public wxObjectWriterCallback
         // this approach would be used if the handler would not
         // be connected really in the designer, so we have to supply
         // the information
-        const wxObject* but = wxAnyGetAsObjectPtr( m_frame->GetProperty("Button") );
+        const wxObject* but = wxAnyGetAsObjectPtr( m_frame->GetProperty(wxString("Button")) );
```

Comment:
Shouldn't this be
```suggestion
        const wxObject* but = wxAnyGetAsObjectPtr( m_frame->GetProperty(wxT("Button")) );
```
?

Reply:
`wxT` works too, I'll update it.

I don't feel like changing all the xti code to accept `wxString` instead of `wxChar` (it's not only properties). Especially since no one enables `wxUSE_EXTENDED_RTTI`.